src/components/VideoCard/VideoCard.jsx

```diff
@@ -0,0 +1,95 @@
+import "./VideoCard.css";
+import { Link } from "react-router-dom";
+import { useState } from "react";
+import { useDataValues } from "../../contextAndReducers/DataProvider";
+import { useWatchLaterActions } from "../../utilities/CustomHooks";
+function MenuCard({ obj }) {
```

Comment:
Keep the MenuCard in a different component folder, cause you will be using it in many places.

Reply:
Yeah gonna do that thanks for noticing it